src/meta/ActiveHostsMan.cpp

```diff
@@ -49,7 +48,7 @@ std::vector<HostAddr> ActiveHostsMan::getActiveHosts(kvstore::KVStore* kv,
         FLOG_ERROR("getActiveHosts failed(%d)", static_cast<int>(ret));
         return hosts;
     }
-    int64_t threshold = (expiredTTL == 0 ? FLAGS_expired_threshold_sec : expiredTTL) * 1000;
+    int64_t threshold = (expiredTTL == 0 ? FLAGS_heartbeat_interval_secs * 2 : expiredTTL) * 1000;
```

Comment:
I think it would be better to increase the time, like 5, the 2 is so small. Or support configurable.

Reply:
Actually, we have been use `FLAGS_heartbeat_interval_secs * 2` in some interfaces for a long time, such as `show hosts`. I believe it is enough.